server/utils.go

```diff
@@ -2,30 +2,19 @@ package main
 
 import (
 	"encoding/json"
+	"fmt"
 	"math/rand"
+	"regexp"
 	"strings"
 	"time"
 
 	"github.com/mattermost/mattermost-server/model"
 )
 
-func (p *Plugin) CreateBotDMPost(userID string, post *model.Post) (*model.Post, *model.AppError) {
-	channel, err := p.API.GetDirectChannel(userID, p.botUserId)
-	if err != nil {
-		p.API.LogError("Couldn't get bot's DM channel", "user_id", userID, "err", err)
-		return nil, err
-	}
-
-	post.UserId = p.botUserId
-	post.ChannelId = channel.Id
-
-	created, err := p.API.CreatePost(post)
-	if err != nil {
-		p.API.LogError("Couldn't send bot DM", "user_id", userID, "err", err)
-		return nil, err
-	}
-
-	return created, nil
+// getServerVersion returns the current server version with only the major and minor version set. For example, both
+// 5.10.0 and 5.10.1 will be returned as "5.10.0" by this method.
+func getServerVersion(serverVersion string) string {
```

Comment:
1/5 that we should use `blang/semver` for this instead of implementing it ourself.

Reply:
I removed the library because this was the only reason we used it, and it didn't seem necessary just for this. I'd rather not add another library for a small function like this, but I'm only 1/5 as well.

Reply:
Fine by me. I was just wondering if there was another reason.